portal-api/src/v1/errors/user-disabled.error.test.js

```diff
@@ -0,0 +1,17 @@
+const UserDisabledError = require('./user-disabled.error');
+
+describe('UserDisabledError', () => {
+  const userId = 'exampleUserId';
```

Comment:
Will there ever be a scenario, where the `userId` will be empty?

Reply:
Nope, we shouldn't ever throw this error as it references a user being blocked (and we need the userId to know the user)